aggregators/dexswap/index.ts

```diff
@@ -0,0 +1,61 @@
+import { FetchOptions } from "../../adapters/types"
+import { exportDexVolumeAndFees } from "../../helpers/dexVolumeLogs";
+import request, { gql } from "graphql-request";
+import { SimpleAdapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import BigNumber from "bignumber.js";
+
+const FACTORY_ADDRESS = '0x3e40739d8478c58f9b973266974c58998d4f9e8b';
+
+const apiKey = '';
+const graphUrl = `https://gateway-arbitrum.network.thegraph.com/api/${apiKey}/subgraphs/id/AaAYom6ATrw15xZq3LJijLHsBj8xWdcL11Xg6sCxEBqZ`;
```

Comment:
you can use our function like this https://github.com/DefiLlama/dimension-adapters/blob/master/fees/thena-v3.ts#L20

Reply:
I don't understand how to use this function, can you explain it to me?